db/seed/generateRecord.js

```diff
@@ -2,7 +2,7 @@ const faker = require('faker');
 
 module.exports = id => (
   {
-    id,
+    id: id.toString(),
```

Comment:
Did you change the id to a string because it's being grabbed from the URL and converting it to a number would be more time-intensive?